### ISSUE TITLE
Colors in `cbsd jls` prevent loop from working

### ISSUE THREAD

**Issue by moviuro** [cbsd/cbsd#364]
FreeBSD version ( uname -a ):

```
% uname -a
FreeBSD castle-of-hearts.xxx.xx 12.0-RELEASE FreeBSD 12.0-RELEASE r341666 GENERIC  amd64
```

CBSD version ( cbsd version ): `12.0.3`

Following code does not work:

```
# for j in $(cbsd jls | awk '{ if (NR != 1) { print $1 }}'); do cbsd jset jname=$j ver=12.0 ; done                                                                                 
No such jail: ampache
[...]
```

Because of https://github.com/cbsd/cbsd/blob/f605bf20a835049e7e7b159290f2234e486e8858/jailctl/jls#L91 :

```
# cbsd jls | awk '{ if (NR != 1) { print $1 }}' | hexdump -C                                       
00000000  1b 5b 30 6d 1b 5b 30 3b  33 36 6d 61 6d 70 61 63  |.[0m.[0;36mampac|
00000010  68 65 0a 1b 5b 30 6d 1b  5b 30 3b 33 36 6d 68 61  |he..[0m.[0;36mha|
[...]
```

Solutions:

* add a switch to disable/enable color

**Comment by olevole** [comment]
Hello @moviuro !  cbsd jset in hard way =) ? How are you ? and cbsd auto-completeion project ;) glad to see you)
It seems that of all the possible options that cdsd offers you, you have chosen the most difficult.

cbsd can use jname as mask for mass-command, so you can do something like:
```
cbsd jset jname='*' ver=native
cbsd jexec jname='jail*' file -s /bin/sh
cbsd pkg jname='myja*l*' mode=install  ca_root_nss nss
cbsd jstop jname='*'
cbsd jstart jname='lala*
```

If you want to display only certain fields in 'cbsd jls', you can use 'header=0' and 'display=<field>', e.g:

`cbsd jls display=jname,ver,astart`

besides this, you can get by one command list of all jails (sorted in boot order sequence) via 'cbsd border'

e.g:

```
for i in $( cbsd border ); do
   echo $i
done
```

And finally you can turn off ANSICOLOR via NOCOLOR environment variable by env or global env (via .cshrc, .bashrc, .login, setenv or export )
try this:

`env NOCOLOR=1 cbsd jls`

or for csh:
```
setenv NOCOLOR=1
cbsd jls
```
or for sh/bash (zfs?):

```
export NOCOLOR=1
cbsd jls

```
or

**Comment by olevole** [comment]
someone also wanted to adjust the color behavior through the configuration file, I think I’ll do it soon

**Comment by olevole** [comment]
global.conf has been added for NOCOLOR settings via config file. 
will be available in the next 12.0.4 release

**Comment by olevole** [comment]
commited to ports tree: 12.0.4
